src/pages/api/stats/calls/date.ts

```diff
@@ -0,0 +1,111 @@
+import { createApiFetch } from 'utils/apiFetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export interface DateStats {
+  date: string;
+  calls: number;
+  conversations: number;
+}
+
+export interface ZetkinCaller {
```

Comment:
Should maybe these two Zetkin... types be in the Zetkin types file?

Reply:
Ideally yes, but I foresee this change a lot with the refactor that I'm proposing, so that's why I didn't prioritize finding a perfect spot for it.

Reply:
Ah, good stuff! keep it :D